actors/builtin/reward/reward_test.go

```diff
@@ -65,5 +65,4 @@ func (h *rewardHarness) constructAndVerify(rt *mock.Runtime) {
 	rt.GetState(&st)
 	emptyMap := adt.AsMultimap(adt.AsStore(rt), st.RewardMap)
 	assert.Equal(h.t, emptyMap.Root(), st.RewardMap)
-	assert.Equal(h.t, big.Zero(), st.RewardTotal)
```

Comment:
Does anyone know why this needs to be removed?

Reply:
RewardTotal is the sum of unwithdrawn but minted funds, RewardActor.balance - RewardTotal is the amount of FIL that has not been minted in supply. We are moving away from the iterative computation for block reward and will send funds to miner actor right after a reward is minted and hence no need for keep tracking of RewardTotal.